src/common/BaseAnnotator.ts

```diff
@@ -101,4 +99,16 @@ export default class BaseAnnotator extends EventEmitter {
     setVisibility = (visibility: boolean): void => {
         this.store.dispatch(setVisibilityAction(visibility));
     };
+
+    addListener(eventName: string, listener: (...args) => void): void {
+        eventManager.addListener(eventName, listener);
+    }
+
+    removeListener(eventName: string, listener: (...args) => void): void {
```

Comment:
What does `(...args)` mean here

Reply:
The listener can take variable arguments